service/chartconfig/v1/helm/spec.go

```diff
@@ -1,11 +1,9 @@
 package helm
 
-import "github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
-
 // Interface describes the methods provided by the helm client.
 type Interface interface {
-	GetReleaseContent(v1alpha1.ChartConfig) (*ReleaseContent, error)
-	GetReleaseHistory(v1alpha1.ChartConfig) (*ReleaseHistory, error)
+	GetReleaseContent(releaseName string) (*ReleaseContent, error)
+	GetReleaseHistory(releaseName string) (*ReleaseHistory, error)
```

Comment:
Unrelated to this PR. I was wondering what again the release name is. Some godoc with examples would be nice.

Reply:
Good point. This gets confusing because both Helm and CNR have concepts named Release. I've added godoc to the interface.